migrations/sqls/20191025023019-lead-status-fix-up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE leads ALTER leadStatus SET DEFAULT "active";
```

Comment:
Do we actually need a db migration for this? Can't this just be something implemented inside of `POST /api/leads`?